.github/workflows/integration.yml

```diff
@@ -187,7 +187,7 @@ jobs:
           #
           # See: https://quarkus.io/guides/tests-with-coverage#coverage-for-integration-tests
           #
-          mvn verify -B --no-transfer-progress \
+          mvn verify -B --no-transfer-progress -pl common,api,operator\
```

Comment:
Why is this needed?

Reply:
The integration runs API and OPERATOR modules, but with the verify and integration in the pom.xml it runs systemtests as well. I thought i should not disable STs by adding a default profile that runs none tests. Could maybe do that using a profile. WDYT? @MikeEdgar

Reply:
You can do it differently. You can disable running the STs for specific goal and then use something else (like mvn clean install vs mvn verify). Or you can create property `skipSTs` and its value can be set for whatever triggers the tests. For example we have `<it.skip>true</it.skip>` and that is the property for Failsafe (or is it Surefire 🤔 ) for skipping the tests. You can play with it a bit. I wouldn't do this, because every time when there will be new module, you will need to remember that you have to add it to this list of modules where you want to run UTs and ITs (and it's easily forgettable).

Reply:
I found it more fitting in this case, but true could skip it here with skip variable i anyway use.